contracts/Controller.sol

```diff
@@ -454,7 +464,26 @@ contract Controller is ReentrancyGuard, Ownable {
      * @dev only account owner or operator can withdraw long option from vault
      * @param _args MintArgs structure
      */
-    // function _mintOtoken(Actions.MintArgs memory _args) internal isAuthorized(_args.owner) {}
+    function _mintOtoken(Actions.MintArgs memory _args)
+        internal
+        isAuthorized(msg.sender, _args.owner)
```

Comment:
we can remove the `msg.sender` and just access it directly in the `isAuthorized` modifier right?

Reply:
Yeah, I thought that can give more idea about it, who is authorized to access what, from the modifier args...

Reply:
Good point